ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -414,6 +444,7 @@ public String build(String dfReferenceId) {
             base.put("challenge_requested", challengeRequested);
             base.put("data_only_requested", dataOnlyRequested);
             base.put("exemption_requested", exemptionRequested);
+            base.put("requested_exemption_type", requestedExemptionType);
```

Comment:
There should be a test for producing this JSON somewhere. It'll be good to get an assertion for sending this parameter.

Reply:
[ThreeDSecureClientUnitTest](https://github.com/braintree/braintree_android/blob/master/ThreeDSecure/src/test/java/com/braintreepayments/api/ThreeDSecureClientUnitTest.java) definitely needs to be refactored, but we can add an assertion to any of the test methods that call `performLookup()`.

Reply:
Ah! Good call. Updated in this commit: https://github.com/braintree/braintree_android/pull/660/commits/a8a58b0b78592a585a2dfe1a91bc86fdf1ec9a82